WalletWasabi.Fluent/Converters/BoolToOpacityConverter.cs

```diff
@@ -0,0 +1,19 @@
+using Avalonia.Data.Converters;
+using System;
+using System.Globalization;
+
+namespace WalletWasabi.Fluent.Converters
+{
+	public class BoolToOpacityConverter : IValueConverter
```

Comment:
in the interest of saving memory this can be implemented like:

https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Data/Converters/StringConverters.cs#L18

and used like

https://github.com/AvaloniaUI/Avalonia/blob/4c620666345993bcd15d12b4f1ed0e82e0d66430/src/Avalonia.Themes.Fluent/TextBox.xaml#L68

Reply:
done